packages/devtools_app/lib/src/screens/logging/logging_screen.dart

```diff
@@ -77,11 +75,11 @@ Example queries:
 
 class _LoggingScreenState extends State<LoggingScreenBody>
     with AutoDisposeMixin, SearchFieldMixin<LoggingScreenBody> {
-  LogData selected;
+  LogData? selected;
 
-  LoggingController controller;
+  LoggingController? controller;
```

Comment:
use initialized pattern and make this late

Reply:
done